### PR TITLE
add recipe for gw_eccentricity

### PR DESCRIPTION
**Pull request by md-arif-shaikh**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by md-arif-shaikh** [comment]
@conda-forge-admin please ping @conda-forge/staged-recipes

File: recipes/gw_eccentricity/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gw_eccentricity-{{ version }}.tar.gz
+  sha256: 90c5c9d4b8beaff26fd0e34414c1a41d6d9f68b8d1a79e07659d69765627cee8
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
```
Comment (ocefpaf) [inline recipes/gw_eccentricity/meta.yaml:15]:
```suggestion
  noarch: python
```

File: recipes/gw_eccentricity/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```
Comment (ocefpaf) [inline recipes/gw_eccentricity/meta.yaml:22]:
```suggestion
    - python >=3.6
    - pip
  run:
    - python >=3.6
```

**Review by ocefpaf — COMMENTED** [review]
Looks like this one can be noarch. Is there a reason to skip Windows?

File: recipes/gw_eccentricity/meta.yaml (hunk 2 above)
Comment (ocefpaf) [reply recipes/gw_eccentricity/meta.yaml:22]: For missing packages it is fine to keep it noarch. That way the package is built only once and, if the missing dependency is ever built on Windows, this one would be ready for Windows already.

Comment (md-arif-shaikh) [reply recipes/gw_eccentricity/meta.yaml:22]: I see. So it is okay if the checks do not pass for Windows for this PR? Also, in our package, the tests are only run for Linux and MacOS and therefore it's not really tested to work for Windows. Do you still think it should use `noarch: python`?

Comment (ocefpaf) [reply recipes/gw_eccentricity/meta.yaml:22]: > I see. So it is okay if the checks do not pass for Windows for this PR?

Yep.

> Also, in our package, the tests are only run for Linux and MacOS and therefore it's not really tested to work for Windows. Do you still think it should use noarch: python?

Unless you do know of anything that is platform specific it is OK. We save valuable CI resources with `noarch` packages.

**Comment by md-arif-shaikh** [comment]
@ocefpaf Made the suggested changes.

**Comment by ocefpaf** [comment]
> [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

We need vijayvarma392 confirmation before we merge this one.

**Comment by vijayvarma392** [comment]
Hi @ocefpaf, looks good to me, and I'm willing to be listed as a maintainer.

**Comment by md-arif-shaikh** [comment]
@ocefpaf ready to be merged?

**Pull request merged**